src/server.c

```diff
@@ -461,7 +461,7 @@ struct redisCommand scriptSubcommands[] = {
      "may-replicate no-script @scripting"},
 
     {"kill",scriptCommand,2,
-     "no-script @scripting"},
+     "no-script allow-busy @scripting"},
```

Comment:
please get the latest from unstable, and add these flags to the json files instead.